more_autograding_examples/cpp_simple_lab_automated_input_output/config/config.json

```diff
@@ -0,0 +1,169 @@
+{
+    "grading_parameters" : {
+        "AUTO_POINTS" : 25,
+        "EXTRA_CREDIT_POINTS" : 2
+    },
+
+    "assignment_message" : "The homework submission area & autograding points for Lab 1 are just practice.<br>The only grades for Lab 1 are the 3 checkpoints recorded by your TA & mentors.",
+
+    "part_names" : [ "README", "CODE" ],
+
+    "testcases" : [
+
+        // *************** README *****************
+        {
+            "type" : "FileCheck",
+            "title" : "README",
+            "actual_file" : "part1/README.txt",
+            "points" : 2
+        },
+
+        // *************** COMPILATION *****************
+        {
+            "type" : "Compilation",
+            "title" : "Compilation",
+            "command" : "clang++ -Wall -o a.out -- part2/*.cpp",
+            "executable_name" : "a.out",
+            "points" : 3
+        },
+
+        // * COMPILATION TESTING view_testcase_message *
+        {
+            "type" : "Compilation",
+            "title" : "Dummy Compilation testing view_testcase_message",
+            "command" : "clang++ -Wall -o a_2.out -- part2/*.cpp",
+            "executable_name" : "a_2.out",
+            "points" : 2,
+            "view_testcase_message" : false
+        },
+
+        // *************** TEST CASES *****************
+        {
+            "title" : "non leap year",
```

Comment:
What are these few testcases actually testing in terms of new functionality? Are we generating this input in some way that I'm not seeing?